src/components/data-explorer/data-explorer.tsx

```diff
@@ -49,6 +49,7 @@ interface DataExplorerDataProps<T> {
     paperProps?: PaperProps;
     actions?: React.ReactNode;
     hideSearchInput?: boolean;
+    currentRoute?: string;
```

Comment:
data-explorer itself has nothing to do with routing - imo paperKey or containerKey is better name for a prop.